### PR TITLE
StarLine integration

### PR DESCRIPTION
**Pull request by Anonym-tsk**
**Description:**
Support for [StarLine](https://www.starline.ru/) car security system.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#27197

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/starline.markdown [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+description: "Instructions on how to setup your StarLine account with Home Assistant."
+logo: starline.png
+ha_category:
+  - Car
+  - Binary Sensor
+  - Presence Detection
+  - Lock
+  - Sensor
+  - Switch
+ha_release: "0.100"
```
Comment (klaasnicolaas) [inline source/_integrations/starline.markdown]:
```suggestion
ha_release: 0.101
```

File: source/_integrations/starline.markdown [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ha_iot_class: Cloud Polling
+ha_qa_scale: gold
```
Comment (klaasnicolaas) [inline source/_integrations/starline.markdown]: @frenck can you check this?

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_integrations/starline.markdown (hunk 2 above)
Comment (frenck) [reply source/_integrations/starline.markdown]: The `ha_qa_scale` is incorrect for this integration. For example, the gold level requires integration tests:

> Tests for reading data from/controlling the integration

Currently, only tests for the configuration flow are provided, hence, it does not meet the gold requirements.

Please adjust the `ha_qa_scale` to a matching level or remove it completely.

For a developers checklist, please see:

<https://developers.home-assistant.io/docs/en/integration_quality_scale_index.html>

File: source/_integrations/starline.markdown [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+- Device tracker: The location of your car.
+- Lock: Control the lock of your car.
+- Sensors: Battery level, SIM card balance, GSM signal level, interior temperature and engine temperature.
+- Switches: Start/stop engine, heater (webasto), additional channel and sound the horn.
+- Services: Update the state, set update frequency. More details can be found [here](/integrations/starline/#services).
+
+## Configuration
+
+1. Create a new application in the [StarLine developer profile](https://my.starline.ru/developer).
+2. Open integrations page and add the StarLine integration.
```
Comment (frenck) [inline source/_integrations/starline.markdown]:
```suggestion
2. Open the integrations page and add the StarLine integration.
```

File: source/_integrations/starline.markdown [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+3. Fill in your application ID, secret, login and password.
+
+After adding the integration, you can configure the API update interval on the integration page.
+
+<div class='note'>
+
+You can make up to 1000 API calls per day which means you could make one approximately every 86 seconds.
```
Comment (frenck) [inline source/_integrations/starline.markdown]:
```suggestion
You can make up to 1000 API calls per day, which means you could make one approximately every 86 seconds.
```

File: source/_integrations/starline.markdown [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+By default, the state of integration will be updated every 3 minutes, making 480 calls per day.
+It is not recommended to set update interval of less than 90 seconds.
```
Comment (frenck) [inline source/_integrations/starline.markdown]:
```suggestion
It is not recommended to set an update interval of less than 90 seconds.
```

File: source/_integrations/starline.markdown [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+The `starline` integration lets you retrieve data of your StarLine security system from the StarLine portal. You will need a working StarLine account.
+
+This integration provides the following platforms:
+
+- Binary Sensors: Hand brake, hood, trunk, alarm status and doors lock state.
+- Device tracker: The location of your car.
+- Lock: Control the lock of your car.
+- Sensors: Battery level, SIM card balance, GSM signal level, interior temperature and engine temperature.
+- Switches: Start/stop engine, heater (webasto), additional channel and sound the horn.
+- Services: Update the state, set update frequency. More details can be found [here](/integrations/starline/#services).
```
Comment (frenck) [inline source/_integrations/starline.markdown]:
```suggestion
- Services: Update the state, set update frequency. More details can be found [here](#services).
```

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there, @Anonym-tsk! 👋 

I've reviewed your PR and left some comments, could you please take a look? Thanks! 👍

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Review by frenck — APPROVED** [review]
Thanks, @Anonym-tsk! 👍

**Pull request merged**